examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -6232,6 +6476,58 @@ endpoint 1 {
     ram      attribute manufacturerExtension default = 255;
   }
 
+  server cluster LaundryWasherMode {
+    callback attribute supportedModes;
+    callback attribute currentMode;
+    callback attribute startUpMode;
+    callback attribute onMode;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute eventList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 2;
```

Comment:
This is still the wrong value, same for the other bits in this .matter file.  Presumably because it was 2 in the .zap, so changing the XML does not help at that point.  Please fix these for all these clusters in all-clusters-app.zap.

Reply:
Fixed [here](https://github.com/project-chip/connectedhomeip/pull/27504/commits/089cc55daf63928a00ec98641fed9e2d868b7331).